packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -74,6 +74,12 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       activeStateEnabled,
       focusStateEnabled,
       hoverStateEnabled,
+      onInput: ({ component }): void => {
+        const { text } = component.option();
+        const isButtonDisabled = !text;
```

Comment:
[missed scenario]
I think we should also keep the button disabled when space character is inputted. Because in this case the message cannot be send => button should not be active

Reply:
QUnit.test('Send button should be disabled after entering spacing into input', function(assert) {
// Test implementation goes here
});

Reply:
I think we can use something simular to consider the same cases when we prevent rise of MessageSend event

https://github.com/marker-dao/DevExtreme/blob/24_2_chat_send_button_disabled/packages/devextreme/js/__internal/ui/chat/chat_message_box.ts#L120-L121

Reply:
[332595f30abb7fcb0abcc63677fca5a99e430962]